### ISSUE TITLE
cannot start a python kernel after a crush

### ISSUE THREAD

**Issue by supersuperpipal** [ipython/ipython#9986]
```
Traceback (most recent call last):
  File "/home/yizhng/anaconda2/bin/ipython", line 6, in <module>
    sys.exit(IPython.start_ipython())
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/IPython/terminal/ipapp.py", line 306, in initialize
    self.init_shell()
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/IPython/terminal/ipapp.py", line 322, in init_shell
    ipython_dir=self.ipython_dir, user_ns=self.user_ns)
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/traitlets/config/configurable.py", line 405, in instance
    inst = cls(*args, **kwargs)
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/IPython/core/interactiveshell.py", line 465, in __init__
    self.init_history()
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/IPython/core/interactiveshell.py", line 1586, in init_history
    self.history_manager = HistoryManager(shell=self, parent=self)
  File "/home/yizhng/anaconda2/lib/python2.7/site-packages/IPython/core/history.py", line 520, in __init__
    self.log.error("Failed to create history session in %s. History will not be saved.",
AttributeError: 'HistoryManager' object has no attribute 'log'
```

please help

**Comment by Carreau** [comment]
This is a duplicate of #9351 which was fixed in #9352 updating your IPython should fix it.

**Comment by ivanov** [comment]
Thank you for your report. I am closing this issue as it is a duplicate and has been fixed.
